library/entity/points_of_interest/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	name = "Quilt Points of Interest"
+	moduleName = "points_of_interest"
+	id = "quilt_points_of_interest"
```

Comment:
I've considered shortening `points_of_interest` to `poi` in the majority of the naming for brevity, but I don't know if that may be too unclear

Reply:
I think its fine for the module names but should be shortened to `POI` in code when it is clear